ui/src/importer/ImporterList.js

```diff
@@ -42,11 +45,57 @@ export const modeFormatter = mode => {
 };
 
 export class ImporterList extends Component {
-  state = {
-    isLoading: true,
-    error: null,
-    data: null,
-  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      error: null,
+      data: null,
+      activePage: 1,
```

Comment:
during the import on the sandbox, the page was jumping back to 1, was it tested?

Reply:
During testing, it worked as intended.... Still that activePage: 1 line worries me so I will take another look on it.

Reply:
That component is not related to that problem.
That activePage is the one of the list of import process (not the one of each import records table).